Android/app/src/main/java/com/example/boardgamesocial/MainApp/Fragments/GameCollectionFragment.java

```diff
@@ -107,6 +126,43 @@ public void onDestroyView() {
 
     @Override
     public void onItemClick(Bundle contextBundle) {
-        Toast.makeText(getContext(),"Game clicked", Toast.LENGTH_LONG).show();
+        //Toast.makeText(getContext(),"Reviews functionality to be implemented!", Toast.LENGTH_LONG).show();
+
+        NavHostFragment.findNavController(GameCollectionFragment.this)
+                .navigate(R.id.action_gameCollectionFragment_to_gameReviews, contextBundle);
+    }
+
+    //Method to remove private games from the games collection
+    private Observable<JsonArray> fetchNonPrivateGames() {
+
+        Log.i(TAG, "Inside fetchNonPrivateGames");
+
+        gameToUserList = new ArrayList<>();
+
+        RetrofitClient.getClient().getCall(GameToUser.class, new HashMap<String, String>() {{
+            put("private", "True");
+        }}).blockingSubscribe(gameToUserJson -> gameToUserList = getObjectList(gameToUserJson, GameToUser.class));
```

Comment:
Function name says fetchNonPrivateGames, but then you set private: True in getCall, did you mean private: False?

Reply:
When a user adds a game to the collection, he has the ability to make that game private, if the user adds a private game then we are storing that information in GameToUser with private: true. So when I fetch using private as true from the GameToUser table it helps me fetch at least those game ids which are true, which later on helps me to exclude from the call where I have fetched all games from the games collection. That ultimately obtains the non-private games.